### PR TITLE
removes health service

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -51,10 +51,6 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/k8scloudconfig"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/giantswarm/k8shealthz"
-
```
Comment (xh3b4sd) [inline Gopkg.toml]: Dropping this now.

File: server/endpoint/endpoint.go [hunk 2]
```diff
@@ -2,44 +2,47 @@ package endpoint
 
 import (
 	"github.com/giantswarm/microendpoint/endpoint/healthz"
-	"github.com/giantswarm/microendpoint/endpoint/version"
-	healthzservice "github.com/giantswarm/microendpoint/service/healthz"
+	versionendpoint "github.com/giantswarm/microendpoint/endpoint/version"
```
Comment (xh3b4sd) [inline server/endpoint/endpoint.go]: The endpoint is aligned with other projects.

File: service/healthz/healthz.go [hunk 3]
```diff
@@ -1,53 +0,0 @@
-package healthz
```
Comment (xh3b4sd) [inline service/healthz/healthz.go]: Gone.

File: service/service.go [hunk 4]
```diff
@@ -42,17 +40,11 @@ type Service struct {
...
 	if config.Flag == nil {
-		return nil, microerror.Maskf(invalidConfigError, "config.Flag must not be empty")
-	}
-	if config.Name == "" {
-		return nil, microerror.Maskf(invalidConfigError, "config.Name must not be empty")
+		return nil, microerror.Maskf(invalidConfigError, "%T.Flag must not be empty", config)
 	}
 	if config.Viper == nil {
-		return nil, microerror.Maskf(invalidConfigError, "config.Viper must not be empty")
+		return nil, microerror.Maskf(invalidConfigError, "%T.Viper must not be empty", config)
```
Comment (xh3b4sd) [inline service/service.go]: Boyscouting.

**Review by kopiczko — APPROVED** [review]

**Pull request merged**